x-pack/plugins/enterprise_search/public/applications/app_search/components/source_engines/source_engines.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiCodeBlock, EuiPageHeader } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { FlashMessages } from '../../../shared/flash_messages';
+import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import { Loading } from '../../../shared/loading';
+import { getEngineBreadcrumbs } from '../engine';
+
+import { SourceEnginesLogic } from './source_engines_logic';
+
+const SOURCE_ENGINES_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.souceEngines.title',
+  {
+    defaultMessage: 'Manage Engines',
```

Comment:
small sentence casing nit
```suggestion
    defaultMessage: 'Manage engines',
```

Also

Reply:
I addressed this in [cc65416](https://github.com/elastic/kibana/pull/96896/commits/cc654166ea55a01918746e7978589df9ca02a644)